### PR TITLE
Add carousel functionality and abstract to feed cards

### PR DESCRIPTION
**Pull request by najinl**
### Features
- Add carousel functionality to news feed

### Refactors 
- Added the abstract to the feed cards
- Commented out box-shadow for articles-display in Feed.css
- Changed the articles-container class flex property from column to row (commented out in Feed.css)
- Deleted divider line between feed cards (commented out in Card.css)

### Issues and Technical Debt
- Because the articles container is changing size as you toggle through the articles in the feed view, the arrows are moving in and out.

### Other Comments, Concerns, or Questions
- NA

### Checklist
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -7,12 +7,13 @@ interface Card {
...
   image: string
   sentiment: number
+  abstract: string
   updateUserSentiment: (userSentiment: number) => void
   storeArticle: (id: string) => void
   topic: string
 }
 
-const Card = ({ title, image, id, shortUrl, sentiment, topic, updateUserSentiment, storeArticle }: Card): JSX.Element => {
+const Card = ({ title, image, id, shortUrl, sentiment, topic, updateUserSentiment, storeArticle, abstract }: Card): JSX.Element => {
```
Comment (anthony-iacono) [inline src/components/Card/Card.tsx:16]: Yes, the abstract will add more value to the feed 👍

File: src/components/Feed/Feed.tsx [hunk 2]
```diff
@@ -37,12 +39,32 @@ const Feed = ({ unreadArticles, updateUserSentiment, storeArticle, selectedTopic
     <>
       <Header />
       <div className="articles-container">
+        <button
+          className="arrow-button"
+          disabled={ articleNumber ? false : true }
```
Comment (anthony-iacono) [inline src/components/Feed/Feed.tsx:44]: Perhaps we can set the arrow to `display: none` when we're at the end of the feed.

**Review by anthony-iacono — APPROVED** [review]
Looks good! We'll smooth out the layout and styles over the next few days.

File: src/components/Feed/Feed.tsx (hunk 2 above)
Comment (najinl) [reply src/components/Feed/Feed.tsx:44]: I like it. Will implement!

**Pull request merged**